src/Template/PrivateMessages/folder.ctp

```diff
@@ -66,7 +66,7 @@ $this->set('title_for_layout', $this->Pages->formatTitle(
                 <h2 flex>
                     <?php
                     $n = $this->Paginator->param('count');
-                    echo format(__n('{folderName} ({n}&nbsp;message)',
+                    echo $this->Number->format($__n('{folderName} ({n}&nbsp;message)',
                                     '{folderName} ({n}&nbsp;messages)',
                                     $n, true),
                                 compact('folderName', 'n'));
```

Comment:
With your modifications, if you try to go to the [private messages locally](http://localhost:8080/eng/private_messages/folder/Inbox), you should see some error in your browser. The page will not be displayed as it should.

1. There isn't supposed to be any `$` after `format(`.
2. You are here mixing the `format()` function with `$this->Number->format()`. These are two different functions. 
    - The `format()` function should not be touched (that is, the line that you modified should not be changed).
    - You want to use `$this->Number->format()` around the number only. Not around the whole text. The number is `$n`.

Now this piece of code is actually a little bit more advanced and probably harder to understand for a beginner because there are functions inside of functions. The functions involved are:

**format()**
This function replaces placeholders inside a string by the value of the associated variable. For instance, if you have `$n = 3` and you do `echo format('I have {n} apples', ['n' => $n])`, it will print `I have 3 apples`.

**__n()**
This function converts the English text into other languages when the UI language is different than English. It handles plural forms of a text.

**compact()**
This function is a short way to create an array. Here for instance, `compact('folderName', 'n')` is just a shortcut for writing `['folderName' => $folderName, 'n' => $n]`.

My hint is: 
1. Change temporarily `$n = $this->Paginator->param('count');` by `$n = 1234`. This will force the text to display "1234 messages" in the private messages page.
2. Replace `compact('folderName', 'n')` by `['folderName' => $folderName, 'n' => $n]`. You should see now difference on the web page.

From there, try to figure out what is left to do so that it displays `1,234 messages` instead of `1234 messages`.
Once you figure it out, don't forget to revert `$n = 1234` back to `$n = $this->Paginator->param('count');`.

Reply:
Uhm... I got this

Error in: ROOT\src\Template\PrivateMessages\folder.ctp, line 69
If you want to customize this error message, create src\Template\Error\error500.ctp

Going to work on it, thank you :)